### PR TITLE
Aidy/create icon in header

### PR DESCRIPTION
**Pull request by aidy8847**

### CODE REVIEW

**Review by ayeletBinder — APPROVED** [review]

File: client-side/src/components/iconHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import EmailIcon from '@mui/icons-material/Email';
+export default function iconHeader(){
+    return(
+        
+        <EmailIcon></EmailIcon>
```
Comment (SariGlick) [inline client-side/src/components/iconHeader.jsx:5]: why do you need this component? you can use @mui/icons-material/Email directly.

File: client-side/src/stories/header/Icon.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Badge, IconButton } from '@mui/material';
+import { MailOutline, Mail } from '@mui/icons-material';
+
+const MessageIcon = ({ messages = [] }) => {
+  const hasMessages = messages.length > 0;
```
Comment (SariGlick) [inline client-side/src/stories/header/Icon.jsx:6]: rename to hasUnreadMessages, and count only unread messages

File: client-side/src/stories/header/Icon.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  const handleClick = () => {
+    //Displays a generic component with the list
+  };
+
+  const iconStyle = { color: 'rgb(103, 252, 210)' };
```
Comment (SariGlick) [inline client-side/src/stories/header/Icon.jsx:12]: move style to scss file, declare color as variable

**Review by SariGlick — CHANGES_REQUESTED** [review]

**Review by SariGlick — CHANGES_REQUESTED** [review]

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "window.autoDetectColorScheme": true
```
Comment (SariGlick) [inline .vscode/settings.json:2]: I see that adding to gitignore after the file was already added to the PR doesn't remove it,
you have to run git rm --cached .vscode/settings.json
in your vscode's terminal, and then commit & push

File: client-side/src/App.jsx [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
-import { store } from './redux/store.jsx';
+import React from 'react';
+import Header from './stories/header/header'
+import Footer from './stories/footer/FooterComponent'
+import DateTimePicker from './components/report';
 import './App.scss';
 
 function App() {
   return (
-    <>
```
Comment (SariGlick) [inline client-side/src/App.jsx:10]: remove your changes from this file, you can copy its content from main branch in git

File: client-side/src/components/report.jsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import GenericButton from '../stories/Button/GenericButton';
+import DateInput from "../stories/DateTime/DateInput";
```
Comment (SariGlick) [inline client-side/src/components/report.jsx:3]: remove this file

File: client-side/src/components/report.scss [hunk 7]
```diff
@@ -0,0 +1,3 @@
+.start {
```
Comment (SariGlick) [inline client-side/src/components/report.scss:1]: remove this file

File: client-side/src/stories/DateTime/DateInput.jsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from 'prop-types';
```
Comment (SariGlick) [inline client-side/src/stories/DateTime/DateInput.jsx:2]: remove this file

File: client-side/src/stories/DateTime/DateInput.scss [hunk 9]
```diff
@@ -0,0 +1,38 @@
+@import '../stylies.scss';
+
+
```
Comment (SariGlick) [inline client-side/src/stories/DateTime/DateInput.scss:3]: remove this file

File: client-side/src/stories/DateTime/DateInput.stories.js [hunk 10]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
```
Comment (SariGlick) [inline client-side/src/stories/DateTime/DateInput.stories.js:2]: remove this file

File: .gitignore [hunk 11]
```diff
@@ -0,0 +1 @@
+.vscode/settings.json
```
Comment (SariGlick) [inline .gitignore:1]: please add also package-lock.json

**Review by SariGlick — CHANGES_REQUESTED** [review]

**Review by SariGlick — APPROVED** [review]

**Review by e171271 — APPROVED** [review]

**Pull request merged**